elastic/kibana
# [Fleet] Fix for bulk update tags, updated API tests

## Summary

Updated tests for bulk actions.

Added one more fix for update tags for the scenario when tags are updated quickly multiple times: https://github.com/elastic/kibana/issues/136092#issuecomment-1259072424
Tested locally with smaller batch size (250) and 1k agents.

To verify:
- bulk select agents (>10k), click Actions and Add / remove tags
- quickly add new tags at least twice
- verify that the second bulk API request contains both new tags
- similarly when removing two tags quickly, the request should contain both tags to remove

https://user-images.githubusercontent.com/90178898/192492806-eed4af93-249c-454e-9a8d-d1893de8e388.mov




### Checklist
